File: src/engine/ChangeLens.Engine/Comparisons/Models/ComparisonRefreshRemoteBaselineResult.cs
namespace ChangeLens.Engine.Comparisons.Models;

/// <summary>
///     Represents the new revision of a refreshed remote-tracking reference in the engine protocol.
/// </summary>
/// <param name="RemoteRevision">The new full object identifier of the refreshed remote-tracking reference.</param>
internal sealed record ComparisonRefreshRemoteBaselineResult(string RemoteRevision);

File: src/desktop/src-tauri/src/comparisons/models/comparison_refresh_remote_baseline_result.rs
use serde::{Deserialize, Serialize};

/// Represents the new revision of a refreshed remote-tracking reference.
#[derive(Clone, Debug, PartialEq, Eq, Deserialize, Serialize)]
#[serde(rename_all = "camelCase", deny_unknown_fields)]
pub struct ComparisonRefreshRemoteBaselineResult {
    /// The new full object identifier of the refreshed remote-tracking reference.
    pub remote_revision: String,
}
